site/content/en/docs/handbook/addons/ingress-dns.md

```diff
@@ -61,6 +61,8 @@ minikube addons enable ingress-dns
 {{% tabs %}}
 {{% linuxtab %}}
 
+## Linux OS with resolvconf
```

Comment:
how about we create a "nested Tab "
```
{{% tabs %}}
{{% withresolveconf %}}...

...
{{% withNetworkManager %}}...
{{% /tabs %}}

Reply:
Hi @medyagh 
it seems like hugo/docsy etc doesn't support embeding tabs into tabs 
![image](https://user-images.githubusercontent.com/3018472/144162444-91f0a171-8e11-4ef3-ae59-d7af489c23af.png)